templates/statistics.hbs

```diff
@@ -0,0 +1,16 @@
+<section class="statistics">
+	<div class="statistics__container">
+		<div class="statistics__headlines">
+			<span class="statistics__eyebrow eyebrow-typography">{{eyebrow}}</span>
```

Comment:
Can you change the `span` for an `h2`? It will improve accessibility

Reply:
done! Thank you :)